corehq/apps/custom_data_fields/templates/custom_data_fields/custom_data_fields.html

```diff
@@ -113,9 +116,9 @@
                                         </span>
                                     </th>
                                     <th class="col-sm-3">
-                                        {% trans "Choices" %}
+                                        {% trans "Validation" %}
                                         <span class="hq-help-template"
-                                              data-title="{% trans "Choices" %}"
+                                              data-title="{% trans "Validation" %}"
                                               data-content="{% blocktrans %}
                                                   Provides a drop-down list of choices for this field
                                                   instead of free text entry.
```

Comment:
Likewise this help text is a bit funny if you have the flag enabled, though I think that's fine if the flag is not on the product pathway.